schema/config-schema.json

```diff
@@ -118,6 +118,10 @@
                 "user": {
                     "id": "https://opencontainers.org/schema/bundle/process/user",
                     "type": "object",
+                    "required": [
+                        "uid",
+                        "gid"
```

Comment:
That being the case, should the `user` field should be removed from the `config-schema`, into different types（config-linux.json/config-windows.json etc）.